server/views/pages/bookedtoday/choosePrisoner.njk

```diff
@@ -16,7 +16,7 @@
 {% block content %}
     <div class="govuk-!-margin-bottom-8">
         <div class="govuk-body">
-            <h1 class="govuk-!-margin-0">{{pageTitle}}</h1>
+            <h1 class="govuk-heading-l">{{pageTitle}}</h1>
```

Comment:
'h1` font size is 38px `govuk-heading-l` is 36px. Amended to ensure we're matching the Design system standards